requirements/edx/base.txt

```diff
@@ -160,7 +160,7 @@ glob2==0.3
 gunicorn==0.17.4
 hash-ring==1.3.1          # via django-memcached-hashring
 help-tokens==1.0.3
-html5lib==0.999
+html5lib==1.0.1
```

Comment:
@a-kryachko 
There were so many same issues with this lib on our installation.
Did you try to build requirements somewhere?

Reply:
- all changes for ORA2 reverted. https://github.com/raccoongang/edx-platform/pull/1594
- bug must be investigated and fixes properly tested.